docs/contributor_guide/index.rst

```diff
@@ -19,6 +19,7 @@ The contribution guide explains how to structure your contributions. Please
    contributing_code
    contributing_example_notebooks
    fairlearn_proposals
+   release
```

Comment:
@riedgar-ms you already have https://fairlearn.org/v0.6.1/contributor_guide/development_process.html#creating-new-releases
Do you have plans of merging the two?

Reply:
I think I should get rid of that one, since we're deprecating the dashboard within Fairlearn itself. I'd forgotten about it entirely :-/